Lesson-7/lesson-7-new/style.css

```diff
@@ -22,933 +22,865 @@
-    margin-left: 128px;
-    width: 98px;
-    height: 19px;
-    top: 2785px;
+    align-items: center;
+    width: 1232px; 
+    height: 46px;
     }
-    .list {
-    display: none;
+    .logo {
+    width: 205px;
+    height: 40px;
+    margin-left: 104px;
+    margin-top: 51px;
     }
-    .container {
-    width: 360px;
-    height: 18px;
-    justify-content: center;
-    margin-left: 30px;
+    h2 {
+        text-decoration: none;
+        color: #383838;
     }
-    .heading_about {
+    h2:hover
+    {
+        text-decoration: underline;
+        color:#F5BD41 
+    }
+    a {
+    text-decoration: none;
+    color: #FFFFFF;
+    display: inline-block;
+    line-height: 1;
+    }
+    a:after
+     {
+    display: block;
+    content: "";
+    height: 3px;
+    width: 0%;
+    transition: width 0.4s ease-in-out;
+    background-color:#F5BD41;
+    }
+    a:hover::after,
+    a:focus::after {
+        width: 100%;
+    }
+    a:hover {
+        color:#F5BD41 ;
+    }
+    .stays {
+    margin-left: 684px;
+    margin-right: 24px;
+    font-family: Roboto;
     font-style: normal;
-    font-weight: 500;
-    font-size: 15px;
-    line-height: 18px;
-    text-align: center;
-    text-decoration-line: underline;
-    }
-    .container_about {
-    margin-left: 67px;
-    margin-right: 16px;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    margin-top: 57px;
+    }
+    .night_img {
+    display: block;
+    width: 30px;
+    height: 30px;
+    margin-top: 46px;
+    margin-left: 48px;
+    }
+    .account {
+    display: block;
+    width: 40px;
+    height: 40px;
+    margin-top: 51px;
+    margin-left: 24px;
+    }
+    .container_middle {
+    width: 1232px;
+    height: 487px;
+    display: flex;
+    flex-direction: column;
+    padding: 202px 104px 120px;
     }
-    .heading_property {
+    .heading {
+    width: 542px;
+    height: 118px;
+    font-family: Roboto;
     font-style: normal;
     font-weight: 500;
-    font-size: 15px;
-    line-height: 18px;
+    font-size: 50px;
+    line-height: 59px;
+    color: #FFFFFF;
+    }
+    label {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
     text-align: center;
-    text-decoration-line: underline;
+    color: #FFFFFF;
+    width: 173px;
+    height: 21px;
     }
-    .container_property_types {
-    margin-right: 16px; 
+    .label {
+    margin-top: 100px;
+    display: flex;
+    margin-bottom: 24px;
     }
-    .container_support {
-    margin-right: 68px;
+    #search {
+    white-space: nowrap;
+    margin-left: 24px;
     }
-    .heading_support {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 15px;
-    line-height: 18px;
-    text-decoration-line: underline;
+    #date {
+    margin-left: 280px;
+    white-space: nowrap;}
+    .place_to_request {
+    display: flex;
+    align-items: stretch;
+    width: 1223px;
+    height: 64px;
```

Comment:
adaptive is not working. Check in your browser how it looks for different screens
![image](https://user-images.githubusercontent.com/20189313/129096325-3d45a2d7-822e-4723-98b6-7ef90735b839.png)